Donut/__init__.py

```diff
@@ -0,0 +1,36 @@
+from flask import Flask, g
+from sqlalchemy import create_engine
```

Comment:
Do we want to continue using 
```
from flask import Flask, g, url_for, render_template, ...
from sqlalchemy import text, create_engine, ...
```
Or do we just want to do this:
```
import flask
import sqlalchemy
...

flask.render_template(...)
etc
```
That means we need to prefix things with `flask.`, but we don't need to worry about keeping the import statement updated with whatever functions/variables we're using.